api/client/samples/crop_w_series/geospatial_data_series_weighted_by_crop_for_region.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<p>It should read <code>2</code>  Crop Acreage, I believe</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>